config/rollup/rollup.config.js

```diff
@@ -147,17 +148,12 @@ function configure(pkg, env, target) {
     }
   }
 
-  if (isModule) {
+  if (isCommonJs) {
     return {
       plugins,
       input,
       onwarn,
       output: [
```

Comment:
instead of this one step doing es and cjs output, it now just does common js output